exonum/src/blockchain/config.rs

```diff
@@ -59,6 +59,8 @@ pub struct StoredConfiguration {
     /// configuration will not take effect at all; the old configuration will
     /// remain actual.
     pub actual_from: Height,
+    /// Core storage version.
+    pub storage_version: u64,
```

Comment:
I propose to remove version from the configuration: it looks like some rudiment, because storage version is our implementation detail and user probably should not be able to choose it.